servicebus/spring-cloud-azure-starter-servicebus-jms/servicebus-jms-queue/README.md

```diff
@@ -108,23 +108,30 @@ Running the command below to export environment values:
 
 ## Run Locally
 
-In your terminal, run `mvn clean spring-boot:run`.
-
-
+Navigate to the project root directory, then run the maven command:
```

Comment:
we can update the template and then update all the descriptions here.
issue [link](https://github.com/Azure-Samples/azure-spring-boot-samples/issues/149)

please revert it here.